tools/monitoring/gke_deployment/monitoring_service.yaml

```diff
@@ -41,11 +41,27 @@ spec:
       containers:
       - name: monitoring-app
         image: gcr.io/knative-tests/test-infra/monitoring:latest
+        command: ["/monitoring",
+                  "--monitoring-database-user=/secrets/cloudsql/monitoringdb/username",
```

Comment:
dont pass this as flags. We can read these directly from vol mounts

Reply:
Removed